benchmarks/scripts/run_strict_supported_set_benchmarks.sh

```
@@ -0,0 +1,375 @@
+jq -n \
+  --slurpfile selected_cases "$selected_cases_json" \
+  --slurpfile rerun "$rerun_json" \
+  --argjson expected_runs "$RUNS" \
+  --argjson expected_cold_runs "$COLD_RUNS" \
+  --arg generated_at "$generated_at" \
+  --arg source_benchmark_json "$SOURCE_BENCHMARK_JSON" \
+  --arg rerun_benchmark_json "$rerun_json" \
+  --arg rerun_benchmark_markdown "$rerun_md" \
+  --arg selection_cases_file "$selected_cases_tsv" \
+  --arg replay_command "$replay_command" \
+  --arg artifact_path "$strict_json" \
```

Shellcheck (0.11.0)

[info] Make sure not to read and write the same file in the same pipeline.

(SC2094)